views/config_postProcessing.mako

```diff
@@ -200,6 +200,18 @@
                                     <span class="component-desc"><b>NOTE:</b> Only working with RAR archive</span>
                                 </label>
                             </div>
+                            <div id="content_unpack">
```

Comment:
@p0psicles need to be that way so if will only show that option if the `unpack` is enabled. But somehow is not working. ping @OmgImAlexis